qlib/QUnit.qm

```diff
@@ -618,7 +618,7 @@ public class QUnit::TestReporter {
             push junit."testsuites"."testcase", testcase;
         }
 
-        printf("%s\n", makeFormattedXMLString(junit));
+        printf("%s\n", make_xml(junit));
```

Comment:
please use make_xml(junit, XGF_ADD_FORMATTING) for the equivalent output to makeFormattedXMLString()